src/utils/converter.utils.ts

```diff
@@ -132,3 +132,14 @@ export const calculateCrc32 = (bytes: Uint8Array): Uint8Array => {
   view.setUint32(0, crc.crc32(Buffer.from(bytes)), false);
   return Buffer.from(checksumArrayBuf);
 };
+
+// 32 bytes
+export const buildSubAccount = (
```

Comment:
This logic is very specific to staking, so I wouldn't put it in a generic utils method.

Reply:
ok, I wasn't sure where to put it.